cluster/plan.go

```diff
@@ -27,9 +27,8 @@ const (
 	ContainerNameLabel = "io.rancher.rke.container.name"
 	CloudConfigSumEnv  = "RKE_CLOUD_CONFIG_CHECKSUM"
 
-	DefaultToolsEntrypoint        = "/opt/rke-tools/entrypoint.sh"
-	DefaultToolsEntrypointVersion = "0.1.13"
-	LegacyToolsEntrypoint         = "/opt/rke/entrypoint.sh"
+	DefaultToolsEntrypoint = "/opt/rke-tools/entrypoint.sh"
+	DefaultRKETools        = "rancher/rke-tools:v0.1.30"
```

Comment:
I think it's better to move this to rancher/types and source it from there.

Reply:
@moelsayed we are working on a bigger task of moving all system images out of types and out of rke :)